pandas/tests/io/test_excel.py

```diff
@@ -2375,36 +2370,50 @@ def style(df):
                           ['', '', '']],
                          index=df.index, columns=df.columns)
 
-    def assert_equal_style(cell1, cell2):
-        # XXX: should find a better way to check equality
-        assert cell1.alignment.__dict__ == cell2.alignment.__dict__
-        assert cell1.border.__dict__ == cell2.border.__dict__
-        assert cell1.fill.__dict__ == cell2.fill.__dict__
-        assert cell1.font.__dict__ == cell2.font.__dict__
-        assert cell1.number_format == cell2.number_format
-        assert cell1.protection.__dict__ == cell2.protection.__dict__
+    pytest.importorskip('jinja2')
+    df = DataFrame(np.random.randn(10, 3))
+    return df.style.apply(style, axis=None)
+
+
+def assert_equal_style(cell1, cell2):
+    # XXX: should find a better way to check equality
+    assert cell1.alignment.__dict__ == cell2.alignment.__dict__
```

Comment:
And I can't recall why right now, but testing for equality of `cell.alignment` did not work.